### PR TITLE
Restrict tests to its namespace

### PR DESCRIPTION
**Pull request by kyessenov**
I suspect we have a starvation issue where controllers overload some resource. This should reduce the load by limiting watchers to the test namespace.
Also fixed:
- marked driver.go manual to avoid building docker images automatically
- tried with recent envoy build and discovered it's broken for manager for unknown reason

### CODE REVIEW

File: test/integration/http-service.yaml.tmpl [hunk 1]
```diff
@@ -43,8 +43,6 @@ spec:
           - "{{.port1}}"
           - --port
           - "{{.port2}}"
-          - --version
```
Comment (ayj) [inline test/integration/http-service.yaml.tmpl]: Intentional? I didn't see anything in the commit summary about removing versioned service.

Comment (kyessenov) [inline test/integration/http-service.yaml.tmpl]: it's not used

Comment (rshriram) [inline test/integration/http-service.yaml.tmpl]: I think if you remove this, the server will throw an error.. Its harmless anyway. Its used by routing tests.

**Comment by kyessenov** [comment]
I did some refactoring in the driver - to allow parallel and sequential requests. I need sequential to understand envoy's problem. Let me know if you have more comments.

File: test/integration/http-service.yaml.tmpl (hunk 1 above)
Comment (kyessenov) [inline test/integration/http-service.yaml.tmpl]: default value for version is "". i don't see any reference to "unversioned" anywhere in the tests.

**Comment by kyessenov** [comment]
I think it's hard to reason about defer with log.Fatal so let's make namespace deletion explicit. This way we know for sure namespaces don't get leaked. I always thought making log.Fatal cause a panic  was a strange decision by golang designers.

**Pull request merged**